lib/hidDevice.js

```diff
@@ -92,8 +93,10 @@ module.exports = function(config) {
       debug('just tried to send nothing!');
     } else {
       var arr = Array.from(buf);
-      arr.unshift(0); // The first byte of arr must contain the Report ID.
-                      // As we only work with a single report, this is set to 0x00.
+      if (config.deviceInfo.driverId !== 'CareSens') {
+        arr.unshift(0); // The first byte of arr must contain the Report ID.
```

Comment:
Why is this different for `CareSens`? Might be nice to have a comment explaining why...

Reply:
Ah, very good point! I'll add a comment.